app/assets/stylesheets/styles/game.scss

```diff
@@ -1,7 +1,9 @@
 #gamesheader {
-  color: #9bb0bb;
+  color: #ffffff;
+  font-weight: 600;
```

Comment:
Properties should be ordered color, font-family, font-weight, margin-left, padding-top